UTRAnnotator.pm

```diff
@@ -14,6 +14,7 @@ limitations under the License.
 
 =head1 CONTACT
 
+    Xiaolei Zhang <xiaolei@ebi.ac.uk>
```

Comment:
Hi @nakib103 - can you remove the email address here. It's better not to advertise personal email addresses to avoid spam.

Reply:
Thanks @sarahhunt, done!